app/src/ui/lib/text-box.tsx

```diff
@@ -288,6 +291,7 @@ export class TextBox extends React.Component<ITextBoxProps, ITextBoxState> {
           onKeyDown={this.onKeyDown}
           tabIndex={this.props.tabIndex}
           onContextMenu={this.onContextMenu}
+          spellCheck={this.props.spellcheck === true}
```

Comment:
As discussed in Slack, seems like the changes in this file shouldn't be needed (since `TextBox` is not used in the commit message box).